modules/statics/src/networks.ts

```diff
@@ -1013,6 +1001,30 @@ class ZkSyncTestnet extends Testnet implements EthereumNetwork {
   walletImplementationAddress = '0x92db2759d1dca129a0d9d46877f361be819184c4';
 }
 
+class BeraChain extends Mainnet implements EthereumNetwork {
```

Comment:
Based on the convention followed in this file, the name of the class and name field are same, so please maintain consistency in case as well
```suggestion
class Berachain extends Mainnet implements EthereumNetwork {
```

Reply:
done